packages/pluginsdk/scripts/scaffold.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+const path = require('path');
+const fs = require('fs');
+const child_process = require('child_process');
+const readlineSync = require('readline-sync');
+
+const pluginNameQuestion = 'Enter the short name for your plugin (default: myplugin): ';
+const pluginName = readlineSync.question(pluginNameQuestion, { defaultInput: 'myplugin' });
+
+const dirQuestion = `Directory to scaffold into (default: ${pluginName}-deck): `;
+const scaffoldTargetDir = readlineSync.question(dirQuestion, { defaultInput: `${pluginName}-deck` });
+
+// Rename these files
+const fileMapping = {
```

Comment:
renaming these files in git to `scaffold.*` Makes it slightly easier to iterate on the scaffold itself.
Without renaming these in git, the tooling (typescript/prettier/IDE) tries to these files as active configuration, and fails because of circular references in configuration composition.  This may not be necessary once `@spinnaker/pluginsdk@0.0.7` is released.